app/templates/play/menu/options-view.pug

```diff
@@ -40,6 +40,15 @@
       span(data-i18n="options.editor_config_behaviors_label") Enable Smart Behaviors
     span.help-block(data-i18n="options.editor_config_behaviors_description") Autocompletes brackets, braces, and quotes.
 
+  if !(features.china)
```

Comment:
there is such a hatred for the editor size that you don't even want to give option 😂

Reply:
haha a unable to get effect option is meaningless and if there's truly someone complain again to ask me give narrow editor back I can update to set default true for china users and enable options. i just don't want to do that for now and get rid of options is easier